app/components/LoginForm/RegisterForm.tsx

```diff
@@ -45,27 +57,42 @@ const RegisterForm = () => {
   }
 
   return (
-    <LegoFinalForm onSubmit={onSubmit} validate={validate}>
+    <TypedLegoForm onSubmit={onSubmit} validate={validate}>
       {({ handleSubmit }) => (
         <Form onSubmit={handleSubmit} onClick={(e) => e.stopPropagation()}>
           <Field
             name="email"
             component={TextInput.Field}
             placeholder="E-post"
           />
-          <Field
-            name="captchaResponse"
-            fieldStyle={{
-              width: 304,
-            }}
-            component={Captcha.Field}
-          />
+          <Field name="captchaResponse" component={Captcha.Field} />
 
           <SubmissionError />
-          <SubmitButton dark>Registrer deg</SubmitButton>
+
+          {spyValues<FormValues>((values) => {
+            if (values.email && isStudMail(values.email)) {
+              return (
+                <ConfirmModal
+                  title="Pass på!"
+                  message="Det ser ut som du prøver å registrere deg med en student-e-post.
+                  Denne kommer du til å miste tilgang til når du går ut fra NTNU, og du vil da også miste brukeren din på abakus.no.
```

Comment:
I'd like some input on the wording here. It's not like the user will immediately lose the account - it's more that you won't see the "deleting account" mails, but I don't want to explain all that and make this text very long.

Reply:
```suggestion
                  Denne kommer du til å miste tilgang til når du går ut fra NTNU, og da vil du ikke lenger kunne lese e-post om oppdateringer som tilbakestilling av passord og automatisk sletting av brukeren din <Tooltip content={"For å respektere personvern og ikke lagre informasjon vi ikke trenger sletter vi brukere som ikke har vært logget inn på ett år. I perioden fram til dette skjer sender vi ut e-post for å advare deg i tilfelle du vil beholder brukeren din."}>InfoIcon</ToolTip>.
```
Not a 100% happy with my seggestion for the wording either, but imo it takes us very few words to add a few bits of context to make the text factually correct so imo we should do just that.

Rubs me the wrong way when people write stuff that's just wrong to avoid people doing something they don't want them to do

Initially my suggestion was without the info tooltip - so might fly without as well - but why not just add it

Reply:
I agree that we shouldn't write anything "wrong", so I've updated it now. Good idea with the tooltip!